### DISCUSSION TITLE
Feature request: EUR support for USD assets

### DISCUSSION THREAD

**Discussion by Krabtus** [ghostfolio/ghostfolio#3786] [Ideas]
Hello everyone,

I would like to suggest a feature that improves the user experience when managing assets. Currently, it is not possible to manage assets that were previously created in the "Administrator - Market Data" area with USD values ​​with amounts in EUR under "Portfolio - Activities".

It would be helpful if users had the option to record these assets in EUR as well. In addition, a feature that allows market data retrieved from Yahoo Finance with USD values ​​to be automatically converted into EUR could be of great benefit to users.

I look forward to hearing your thoughts on the idea!

**Comment by dtslvr** [comment]
Hi @Krabtus

Thanks for starting this discussion! It seems related to issue https://github.com/ghostfolio/ghostfolio/issues/3320, so I'll close this discussion for better tracking.
